### PR TITLE
publish topics integration guide

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:
- publish topics integration guide
- staged: https://pr-6483-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/topics/integration-guide/

### CODE REVIEW

**Comment by samdutton** [comment]
One small thing: I wonder if subhead could be more factual: 'Learn how to work with the API, from end to end.' is fine, but it doesn't really tell me what this doc is for.

Maybe something like 'Learn how to test the Topics API for ad tech use cases.' or 'Learn how to use the Topics API to meet ad tech use cases.'

**Comment by samdutton** [comment]
Also wondering if we could have an aside at the top, specifically asking for feedback. Something like this:

{% Aside %}
We'd love to know how this article could be improved! 

Follow the [Improve article](https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/en/docs/privacy-sandbox/topics/integration-guide/index.md) link at the foot of the page to make suggestions.
{% endAside %}

Maybe... Not sure.

**Review by samdutton — APPROVED** [review]

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:6]:
```suggestion
date: 2023-06-02
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:68]:
```suggestion
1. Local testing and deployment (Estimated time: around 2 days)
```

Comment (rachelandrew) [reply site/en/docs/privacy-sandbox/topics/integration-guide/index.md:68]: Use words for things like this.

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:90]:
```suggestion
{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/4LysAisUZpkryMHwk18f.png", alt="Take a look inside the Topics API at chrome://topics-internals.", width="768", height="669" %}
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:98]:
```suggestion
* **Topics State:** Display topics observed for the current user.
* **Classifier:** Preview topics inferred for hostnames.
* **Features and Parameters:** View API parameter values to check that feature flags are working as intended.
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:362]: The following set of sentences don't have periods at the end, can you add them?

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:370]:
```suggestion
2. [Optional] Use Topics data as a contextual signal (with URLs, tags, etc.) (Estimated time: around 3 days)
```

File: site/en/docs/privacy-sandbox/topics/integration-guide/index.md
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/topics/integration-guide/index.md:453]:
```suggestion
1. Process and use topics data (Estimated time: around 4 weeks).
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Just a few nits in this. Missing punctuation etc. Ping me when fixed.

**Comment by samdutton** [comment]
Added a DO NOT MERGE label pending review from Leeron (see Chat).

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**
